README.md

```diff
@@ -24,28 +24,27 @@ Any changes made by this module affect the current `running-config`. These chang
 
 ## Setup
 
-### Setup Requirements
+**Device Connection:** This module requires a user that can access the device via SSH and that has the `enable mode` privilege.
 
-This module requires a user that can access the device via SSH and that has the `enable mode` privilege.
+**Proxy Agent Node:** Since there is no agent available for the Catalysts (and, seriously, who would want to run an agent on them?) we need to run puppet on a proxy agent node (either a compile master, or regular puppet agent install).
```

Comment:
```suggestion
**Proxy Agent Node:** Since a Puppet agent is not available for the Catalysts (and, seriously, who would want to run an agent on them?) we need a proxy Puppet agent (either a compile master, or another agent) to run puppet on behalf of the device.
```

Reply:
Proxy Puppet Agent = Proxy Puppet agent